pyproject.toml

```diff
@@ -33,15 +33,14 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
     "Topic :: Software Development",
     "Topic :: Software Development :: Libraries",
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Typing :: Typed",
 ]
 
 [tool.poetry.dependencies]
-python = ">= 3.8"
+python = ">= 3.8 < 3.11"
```

Comment:
We shouldn't set an upper version for Python. We've had issues with it before on Connexion. See:
- https://github.com/python-poetry/poetry/issues/3747
- https://stackoverflow.com/questions/73116647/why-cant-i-install-a-python-package-with-the-python-requirement-3-8-3-11-i